server/bundler/i18n-calypso-loader.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function( content ) {
+	this.cacheable && this.cacheable();
+
+	if ( this.resource.endsWith( 'node_modules/i18n-calypso/lib/index.js' ) ) {
+		const newContent = content.replace(
```

Comment:
Just a question, I'm probably missing something obvious, why do we need to do that via loader and not put that directly into [i18n-calypso](https://github.com/Automattic/i18n-calypso) ?

Reply:
Because we generate Webpack code that will be translated again by Webpack to:
```	
    // Block injected at build time via i18n-calypso-loader Webpack loader
	if ( this.state.localeSlug === 'en' ) {
		// Moment ships with the English locale by default, no need to load anything in that case
	} else {
		__webpack_require__("./node_modules/moment/locale recursive ./node_modules/bundle-loader/index.js?name=moment-locale-[name]!./ ^\\.\\/.*$")("./" + this.state.localeSlug)( () => {
			moment.locale( this.state.localeSlug );
		} );
	}
```

Reply:
Aha, right, thank you for explaining!

Reply:
Also we may want to use `i18n-calypso` without Webpack ;).